### PR TITLE
Fixes #12573: Api Documentation righting

### PR DESCRIPTION
**Pull request by Haypierre**
https://www.rudder-project.org/redmine/issues/12573

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ curl -X POST -H <span class="str">"X-API-Token: yourToken"</span> http://rudder.
 
 Instead of passing parameters one by one, you can instead supply a JSON object containing all you want to do. You'll also have to set the *Content-Type* header to **application/json** (without it the JSON content would be ignored).
 
-The supplied file should be valid JSON: strings need quotes, booleans and integers don't, ...
+The supplied file must contains a valid JSON object: strings need quotes, booleans and integers don't, ...
```
Comment (amousset) [inline introduction.md]: contain

Comment (VinceMacBuche) [reply introduction.md]: Must be a valid JSON (no need to add object)

File: src/api_v2/rules.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
     @apiVersion 2.0.0
     @apiName ruleDetails
     @apiGroup Rules
-    @apiDescription Get the details of one Rule, whose ID is specified in the URL
+    @apiDescription Get details for one rule by specifying its ID in the URL
```
Comment (VinceMacBuche) [inline src/api_v2/rules.md]: Uppercase R for Rules (same for Directives/Techniques/Groups)

File: introduction.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ to keep your existing scripts (based on previous behavior). Versions will always
 
 The versions start with version 2, since version 1 used to be a non-REST basic API for internal purposes. A notable exception is the *latest* version, which will always point to the most recent version available.
 
-It is possible to select the API version you want to use in:
+You may want to select the API version to call:
```
Comment (VinceMacBuche) [inline introduction.md]: You can change the version of the API used by setting it either within the url or in a header (see below)

**Pull request merged**